### ISSUE TITLE
Assembly small DNA fragments (amplicons)

### ISSUE THREAD

**Issue by lborcard** [marbl/canu#2269]
Dear Canu team,

We are working with small DNA fragments (400bp > ,< 5kb)coming from amplicons and we do not always have a reference to align them  and call a consensus. In certain cases we need to assemble them de novo. We have tried to assemble these small fragments with Canu but it seems that we cannot find the right parameters to obtain a correct contig. Do you know if your software can assemble small DNA fragments? If so could you recommend parameters that we should tweak to make it work?

best regards,

Loïc

**Comment by lucyintheskyzzz** [comment]
Hi Loic,
I ended up using these parameters for my low reads I published here: https://www.frontiersin.org/articles/10.3389/fmicb.2023.1248323/full

"Canu 2.2 program parameters for low coverage reads were genomeSize = 2 m maxinputCoverage = 10,000 corOutCoverage = 10,000 corMhapSensitivity = high corMinCoverage = 0 redMemory = 32 oeaMemory = 32 batMemory = 64 minInputCoverage = 0 stopOnLowCoverage = 0, and genomeSize = 2 m maxInputCoverage = 100 –nanopore fecal samplesDNA.fastq for high coverage data, respectively."

**Comment by skoren** [comment]
Generally, yes, amplicons can be assembled. I'm not sure what you mean by "the parameters to obtain a correct contig"? Did you get an assembly but it didn't match your expectation or did you not get an assembled single sequence? The FAQ has some suggestions on tweaking parameters, https://canu.readthedocs.io/en/latest/faq.html#can-i-assemble-amplicon-sequence-data. You can also try the above.

**Comment by lborcard** [comment]
Thank you for your answer @lucyintheskyzzz , you said you used this for low coverage reads but I have the opposite since I am working with amplicons. 
@skoren Sorry for beeing unclear, I was benchmarking my assembly using a ref based appraoch and the assembly was never really of the expected size . And I also obtained several contigs . I was wondering if I could also use a `genomeSize` below 1000 bp?   
Thank you for the link, I read the parameters documentation extensively as well as your publication but I did not think of looking at the FAQ.

thank you so much
